maintenance/projects/js-toolkit/packages/liferay-npm-bundler-plugin-replace-browser-modules/src/index.ts

```diff
@@ -124,6 +124,9 @@ function processAliases(
 					).linkToCode(2);
 					break;
 				}
+
+				default:
+					break;
```

Comment:
We could throw an error here for an unhandled case, but I don't know all the cases and prefer not to get side-tracked looking for them at this point. We can always loop back to this and do an audit, and in the meantime, we are probably not making things worse by doing this... I think... the alternative would be to add a suppression, which at least would give us something to grep for later on. 🤷